policies/project-directorship/board-meeting-substitutes.md

```diff
@@ -0,0 +1,13 @@
+# Rust Foundation Board Meeting Substitutes
+
+The [Rust Foundation bylaws](https://foundation.rust-lang.org/policies/bylaws) do not give any instructions on how a substitute for a Project Director who cannot make a Foundation Board meeting should be selected. Only the following is state in section 4.3(h):
+
+> Each Director may designate in writing or by electronic transmission to the Chairperson or Secretary (which designation may be withdrawn in writing at any time by such Director or Member) an individual to act as a Director in their stead, whether for a single meeting or as a standing alternate. Any such alternate Director shall be entitled to (i) attend and vote at all meetings which the designating Director does not attend, (ii) sign all written consents in lieu of the designating Director, and (iii) otherwise exercise the duties and enjoy the privileges of the designating Director in the absence or unavailability of the designating Director; provided, however, that no such alternate Director may propose a vote or vote upon any Committee of the Board.
+
+The Rust project, however, sets the following policy for how a substitute for a Project Director is determined:
+
+* The Project Director should inform the other Project Directors and the Council of their absence as soon as possible preferably at least 7 days before the Foundation board meeting will take place.
+* The absence of the Project Director may be announced publicly.
+* Any Project Director or Council member may nominate any Project member as a potential substitute unless the member has an open moderation sanction against them.
```

Comment:
We've had similar policies in other places that we might want to formalize. Effectively we want to prevent a team member who is not in good standing from being added as a sub.

I agree that moderation sanctions would not and should not be public knowledge, but it seems particularly draconian for it to be possible for there to be moderation sanctions that only the mod team and the individual in question know about. It doesn't seem inappropriate for the Council to be made aware that there is a moderation investigation against an individual as long as the details aren't revealed and none of this is made public.

Reply:
Perhaps:

> Any Project Director or Council member may nominate any Project member as a potential substitute provided the nominated member is in good standing with the project (e.g. they do not have an open moderation sanction against them).

I wouldn't want to tie this policy directly to moderation action disclosure rules, so hopefully this gives us some flexibility whilst making it clear we're stepping around the obvious loophole. And in anycase, I hope we can trust the PDs and Council members to come to a sensible consensus here.